pkg/release/condition_set.go

```diff
@@ -120,6 +120,32 @@ func (c *conditionSet) PodExists(ctx context.Context, namespace, labelSelector s
 	}
 }
 
+func (c *conditionSet) PodNotFound(ctx context.Context, namespace, labelSelector string) ConditionFunc {
+	return func() error {
+		o := func() error {
+			pods, err := c.k8sClient.CoreV1().Pods(namespace).List(metav1.ListOptions{LabelSelector: labelSelector})
+			if apierrors.IsNotFound(err) {
+				return nil
+			} else if err != nil {
+				return microerror.Mask(err)
+			}
+			if len(pods.Items) != 1 {
+				return microerror.Maskf(waitError, "expected 1 Pod but got %d", len(pods.Items))
+			}
+
+			return microerror.Maskf(waitError, "Pod for label selector %#q in namespace %#q still exists", labelSelector, namespace)
```

Comment:
Here you still use the namespace in the Maskf

Reply:
Pod is namespaced :)